omnibus/config/software/chef.rb

```diff
@@ -69,13 +69,6 @@
   block { log.info(log_key) { "" } }
   bundle "install --verbose", env: project_env
 
-  # For whatever reason, nokogiri software def deletes this (rather small) directory
```

Comment:
pretty certain this was done to avoid files-with-spaces which i think still breaks either aix or solaris 10 builds.